server/mcp_server.py

```
@@ -140,28 +153,79 @@ def _get_token() -> str:
-        except httpx.HTTPStatusError as exc:
-            code = exc.response.status_code
-            try:
-                detail = exc.response.json()
-            except Exception:
-                detail = {"error": exc.response.text[:500]}
-            raise ValueError(f"Aurora API returned {code}: {detail}")
-        return resp.json()
+            detail = exc.response.json()
+        except (ValueError, httpx.ResponseNotRead):
+            detail = {"error": exc.response.text[:500]}
+        raise ValueError(f"Aurora API returned {code}: {detail}") from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)